Formula/mesos.rb

```diff
@@ -66,32 +66,6 @@ def install
     (lib/"python2.7/site-packages").mkpath
     (lib/"python2.7/site-packages/homebrew-mesos-boto.pth").write "#{boto_path}\n"
 
-    # work around distutils abusing CC instead of using CXX
-    # https://issues.apache.org/jira/browse/MESOS-799
-    # https://github.com/Homebrew/homebrew/pull/37087
-    native_patch = <<-EOS.undent
-      import os
-      os.environ["CC"] = os.environ["CXX"]
-      os.environ["LDFLAGS"] = "@LIBS@"
-      \\0
-    EOS
-    inreplace "src/python/native/setup.py.in",
-              "import ext_modules",
-              native_patch
```

Comment:
they mention that the relevant python bug is still open
https://bugs.python.org/issue1222585

On Tue, Aug 2, 2016 at 6:04 PM, ilovezfs <notifications@github.com> wrote:

> In Formula/mesos.rb
> <https://github.com/Homebrew/homebrew-core/pull/3538#discussion_r73265881>
> :
>
> > @@ -66,32 +66,6 @@ def install
> >      (lib/"python2.7/site-packages").mkpath
> >      (lib/"python2.7/site-packages/homebrew-mesos-boto.pth").write "#{boto_path}\n"
> >
> > -    # work around distutils abusing CC instead of using CXX
> > -    # https://issues.apache.org/jira/browse/MESOS-799
> > -    # https://github.com/Homebrew/homebrew/pull/37087
> > -    native_patch = <<-EOS.undent
> > -      import os
> > -      os.environ["CC"] = os.environ["CXX"]
> > -      os.environ["LDFLAGS"] = "@LIBS@"
> > -      \\0
> > -    EOS
> > -    inreplace "src/python/native/setup.py.in",
> > -              "import ext_modules",
> > -              native_patch
>
> I'll try with that put back in. Any idea why upstream seems to think it is
> resolved? https://issues.apache.org/jira/browse/MESOS-799
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Homebrew/homebrew-core/pull/3538/files/30732e17f793e2e007fda863f15cba7c272e53ca#r73265881>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAKnQYQ4D0LLVFdr_kty7JmmLuvJTpzCks5qb-kKgaJpZM4Jaskn>
> .
>



-- 
Tim Smith
Biomedical Engineering
Liu Lab <http://liu.bme.uci.edu>, University of California, Irvine
http://tim-smith.us, twitter: @biotimylated
<http://twitter.com/biotimylated>